cuprum/unittests/test_fetch_main_benchmark_baseline.py

```
@@ -443,3 +443,56 @@ def fake_load_json_response(*, url: str, token: str) -> cabc.Mapping[str, object
+    )
+
+    find_latest_artifact_download_url(
+        query=ArtifactQuery(
+            repository="leynos/cuprum",
+            workflow="ci.yml",
+            branch="main",
+            event="push",
+            artifact_name="benchmark-ratchet-main-baseline",
+            run_status="completed",
+        ),
+        token="".join(("tok", "en")),
```

Ruff (0.16.2)

[warning] Consider `"token"` instead of string join

Replace with `"token"`

(FLY002)